daemon/src/maker.rs

```diff
@@ -29,7 +29,7 @@ use xtra::Actor;
 
 mod routes_maker;
 
-#[derive(Clap)]
+#[derive(Parser)]
```

Comment:
imho this would be nicer if we fully qualify this name in all places, ie. `#[derive(clap::Parser)]` - otherwise it's not that obvious what Parser is it (meanwhile, the previous name was literally the name of the package)